syntax = "proto3";

package protos;
option go_package = "./proto";

import "protos/message.proto";


// 定义交易履约系统服务接口
service Java_A {
  rpc Java_A_1(CommonMessage) returns (CommonMessage) {}
  rpc Java_A_2(CommonMessage) returns (CommonMessage) {}
  rpc Java_A_3(CommonMessage) returns (CommonMessage) {}
}
